scan

```diff
@@ -373,7 +374,14 @@ def perform_scan(params):
 # Let all errors bubble up to perform_scan.
 def perform_local_scan(scanner, domain, handles, environment, options, meta):
     logging.warn("\tExecuting local scan...")
-    response = scanner.scan(domain, environment, options)
+
+    # Special Python->JS shim for local use of headless Chrome.
+    if hasattr(scanner, "scan_headless") and (scanner.scan_headless is True):
+        response = headless_scan(handles[scanner]['name'], domain, environment, options)
+
+    # Otherwise, just call out and expect the scan to run in Python.
+    else:
+        response = scanner.scan(domain, environment, options)
```

Comment:
The `perform_local_scan` function is now bifurcated - if a scanner has `scan_headless` set to `True`, then it is not expected/required to have its own `scan()` function. Instead, the central scanner will import and use `scanners/headless/local_bridge.py`'s general scan function and pass the name of the scanner to it, and `local_bridge.py` will shell out to Node from there. So only the Node scanner (e.g. `scanners/third_parties.js`) is expected to have a `scan()` function.